tests/Policy_1.py

```diff
@@ -112,7 +116,7 @@ def testPolicySetget(self):
 
     def testSimpleLoad(self):
         # n = mwid.Citizen_census(0)
-        p = Policy.createPolicy("examples/EventTransmitter_policy.paf")
+        p = Policy.createPolicy(os.path.join(proddir, "examples", "EventTransmitter_policy.paf"))
```

Comment:
This may be out of scope (in which case please submit a new ticket), but I think test data should not be in the examples directory. Preferably the examples would look in tests/data if they really need to share.